.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       run: |
           sudo apt-get update
           sudo apt-get -y install uuid-dev libcurl4-openssl-dev
-          make KEYDB_CFLAGS='-Werror' KEYDB_CXXFLAGS='-Werror' BUILD_TLS=yes -j2
+          make REDIS_CFLAGS='-Werror' REDIS_CXXFLAGS='-Werror' BUILD_TLS=yes -j2
```

Comment:
The correct fix here is to standardize on KEYDB_CXXFLAGS /CFLAGS not regress back

Reply:
I considered that but figured we didn't want to break existing builds, should have just mentioned it to you. I'll change it now.